src/components/Signup/Signup.js

```diff
@@ -112,6 +112,19 @@ const Signup = function ({
 
   const utmCookies = manageUtmCookies(localStorage, utmSource, utmCampaign, utmMedium, utmTerm);
 
+  useEffect(() => {
+    const script = document.createElement('script');
+  
+    script.src = 'https://crm.zoho.com/crm/javascript/zcga.js';
+    script.async = true;
+  
+    document.body.appendChild(script);
+  
+    return () => {
+      document.body.removeChild(script);
+    }
+  }, []);
```

Comment:
We are using [Helmet](https://github.com/nfl/react-helmet) for this kind of thing.